src/components/share-button/share-button.component.html

```diff
@@ -1,2 +1,12 @@
-<button (click)='share()' [innerHTML]="button.template" [ngClass]="button.classes"></button>
-<span *ngIf="count && shareCount" class="sb-button-count">{{ shareCount | nFormatter: 1 }}</span>
\ No newline at end of file
+<button  [innerHtml]="button.template" [ngClass]="button.classes"
+[shareButton]="button.provider"
+[sbUrl]="url"
+[sbImg]="image"
+[sbTitle]="title"
+[sbDesc]="description"
+[sbTags]="tags"
+[sbCount]="count"
+(sbCount$)="counter($event)"
```

Comment:
still not convinced about that `$` suffix... We use `()` in angular to bind to outputs. That should be enough...